### ISSUE TITLE
Get-LatestUpdate does not return latest CU

### ISSUE THREAD

**Issue by bobac** [aaronparker/LatestUpdate#19]
# Expected Behavior

Get-LatestUpdate should return latest Cumulative Update as per desired Windows Version and Build. So I'd expect that for example for Windows 2016 Server (-WindowsVersion Windows10 -Build 14393) it should return packages with KB4457127 right now (October 9th, 2018 - check https://support.microsoft.com/en-us/help/4457127).

Based on https://support.microsoft.com/en-us/help/4099479/windows-10-update-history
# Current Behavior
Get-LatestUpdate returns KB4343898 as of 2018-08.

# Failure Information (for bugs)

```
PS C:\Users\rhouser> Get-LatestUpdate -WindowsVersion Windows10 -Build 14393 -Verbose
VERBOSE: Adding variable for dynamic parameter 'Build' with value '14393'
VERBOSE: Check updates for 14393 Cumulative.*x64
VERBOSE: Downloading https://support.microsoft.com/app/content/api/content/asset/en-us/4000816 to retrieve the list of
updates.
VERBOSE: GET https://support.microsoft.com/app/content/api/content/asset/en-us/4000816 with 0-byte payload
VERBOSE: received 24901-byte response of content type application/json; charset=utf-8
VERBOSE: Found ID: KB4343887
VERBOSE: GET http://www.catalog.update.microsoft.com/Search.aspx?q=KB4343887 with 0-byte payload
VERBOSE: received 53940-byte response of content type text/html; charset=utf-8
VERBOSE: Ids found:
VERBOSE:  cb91052d-37e8-4caa-bcd8-ec8c84a6aea1
VERBOSE:  0ff8a4be-bcf5-443a-9805-9c5e2f04ab09
VERBOSE:  a746522b-74fd-4a74-a86c-5e45eb1b86df
VERBOSE:  cffa7f5a-126a-48d4-9cb6-5541b12e862e
VERBOSE:  348e720d-750f-481e-b0e0-58bab6ce8538
VERBOSE:  c796509e-8bbb-4de4-b45f-92de37e32d36
VERBOSE: innerText found. Parsing KB notes
VERBOSE: Download cffa7f5a-126a-48d4-9cb6-5541b12e862e
VERBOSE: POST http://www.catalog.update.microsoft.com/DownloadDialog.aspx with -1-byte payload
VERBOSE: received 32188-byte response of content type text/html; charset=utf-8
VERBOSE: Download 348e720d-750f-481e-b0e0-58bab6ce8538
VERBOSE: POST http://www.catalog.update.microsoft.com/DownloadDialog.aspx with -1-byte payload
VERBOSE: received 32153-byte response of content type text/html; charset=utf-8

KB        Note                                                                                     URL
--        ----                                                                                     ---
KB4343887 2018-08 Cumulative Update for Windows 10 Version 1607 for x64-based Systems (KB4343887)  http://download.w...
KB4343887 2018-08 Cumulative Update for Windows Server 2016 for x64-based Systems (KB4343887)      http://download.w...
```

## Steps to Reproduce

Just invoke ```Get-LatestUpdate -WindowsVersion Windows10 -Build 14393```

## Context

* **LatestUpdate Version**: 1.1.0.38
* **PowerShell Version**: 5.1.14393.2515
* **Operating System**: Windows 2016 Server

**Comment by BladeFireLight** [comment]
this is the same as issue 17

**Comment by aaronparker** [comment]
Fixed in commit 72b63c0a8577f10e14d090ea4dc6732eb1c7b7cf
